tasks/build/Gaia/localBuild.sh

```diff
@@ -1,14 +1,23 @@
 #!/bin/sh
 
 . ./COMMIT.sh
+version="$(pwd)/../../../builds/GaiaVersions/$COMMIT"
 source="$(pwd)/../../../builds/Gaia"
-mkdir -p "$source"
-export TARGET=/mnt
+mkdir -p "$version"
+rm -rf "$source"
+if [[ -f "$version/linux/gaiad" ]]; then
+    echo "Already built"
```

Comment:
do we need to be able to rebuild existing versions?

Reply:
you could just remove the existing version from your local file system. should be an edge case

Reply:
if we need it we can add a hook